### ISSUE TITLE
Add 64 bit image builds separately

### ISSUE THREAD

**Issue by freerainboxbox** [melkortf/tf2-servers#145]
[April 18th, 2024 TF2 Update adds official 64-bit builds for both client and server](https://www.teamfortress.com/post.php?id=220436)

Build pipeline should have separate tags for 64-bit builds.

There's currently little to no documentation on how to run the server in 64-bit mode, but this issue should be watched.

**Comment by garrappachc** [comment]
Yup, base image can be already built, but sourcemod-based images need to have proper [DHooks support](https://github.com/alliedmodders/sourcemod/issues/2114)

**Comment by garrappachc** [comment]
Support for the tf2-base image is added in https://github.com/melkortf/tf2-servers/pull/158
